schema/sql/nss.sql

```diff
@@ -11,7 +11,6 @@ create view nss_pgsql.groupmember as
         aux_groups.gid as gid
         from public.passwd inner join public.aux_groups
             on (passwd.uid = aux_groups.uid);
-grant select on nss_pgsql.groupmember to nss_pgsql;
```

Comment:
@daurnimator will dropping these grants have any ill affects on the nss_pgsql user we may be using it for?